mujoco_ros/scripts/keyboard_service.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# license removed for brevity
+import sys,tty,termios
+import curses
+
+import rospy
+from mj_models.msg import MushrControl
+from mj_models.msg import MushrObservation
+from mj_models.srv import MushrFeedback,MushrFeedbackResponse
+
+screen = None
+control = MushrControl()
+observation = MushrObservation()
+robot_name = "mushr"
+velocity_increase = 0.1
+steering_increase = 0.1
+
+def handle_service(req):
+    global control
```

Comment:
I am not a fan of Python `global`. It makes sense to put all of these in a Python class. Let me work on that.